src/Microsoft.ML.StandardLearners/StandardLearnersCatalog.cs

```diff
@@ -219,10 +248,18 @@ public static class StandardLearnersCatalog
         }
 
         /// <summary>
-        /// Predict a target using a linear binary classification model trained with the AveragedPerceptron trainer.
+        /// Predict a target using a linear binary classification model trained with averaged perceptron trainer using advanced options.
+        /// For usage details, please see <see cref="AveragedPerceptron(BinaryClassificationCatalog.BinaryClassificationTrainers, string, string, string, IClassificationLoss, float, bool, float, int)"/>
         /// </summary>
```

Comment:
i would prefer we have full text on all extension, and not redirect ppl to the other one just to read a paragraph..

Reply:
as discussed offline, moving this text to the estimator class

---
In reply to: [256252156](https://github.com/dotnet/machinelearning/pull/2517#discussion_r256252156) [](ancestors = 256252156)